### PR TITLE
VZ-9148: BFS of rancher fleet v0.6.0

### PR DESCRIPTION
**Pull request by Abhijithang**
This PR builds from source of rancher fleet v0.6.0 to support rancher release v2.7.2.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+	$(GO) install github.com/verrazzano/rancher-dapper@${DAPPER_VERSION}
+	mv ${DAPPER_PATH}/bin/rancher-dapper $(DAPPER_PATH)/bin/dapper
+	$(eval DAPPER=$(DAPPER_PATH)/bin/dapper)
+else
+	$(eval DAPPER=$(shell command -v dapper))
+endif
+
+serve-docs: mkdocs
+	docker run -p 8000:8000 --rm -it -v $${PWD}:/docs mkdocs serve -a 0.0.0.0:8000
+1
```
Comment (mgianatagh) [inline Makefile]: Is this "1" an extra character?

Comment (Abhijithang) [reply Makefile]: Apologies! Having a bad day. Fixed it. It was a mistake.

**Review by mgianatagh — APPROVED** [review]

File: Dockerfile.dapper [hunk 2]
```diff
@@ -1,10 +1,19 @@
-FROM registry.suse.com/bci/golang:1.19-20.4
+FROM ghcr.io/oracle/oraclelinux:7-slim
```
Comment (prasad-shirodkar) [inline Dockerfile.dapper:1]: Shouldn't this be oraclelinux:8-slim instead? Same applies to other dockerfiles in this PR.

Comment (Abhijithang) [reply Dockerfile.dapper:1]: Not sure when we need to update the base image to 8-slim. I just used the OL7 similar to others.

Comment (prasad-shirodkar) [reply Dockerfile.dapper:1]: I believe it was decided to use oraclelinux:8-slim going forward.

Comment (Abhijithang) [reply Dockerfile.dapper:1]: @mgianatagh Do we need to migrate all BFS images to OL8 in this task? I believe that would be a different effort right?
I am happy to do it but just checking with you.

Comment (Abhijithang) [reply Dockerfile.dapper:1]: I checked with @mgianatagh and it appears we would need an epic to move all of our BFS builds to move to OL8. So right now Ol7 will stay for a while.

**Pull request merged**